Manifest.toml

```diff
@@ -1,20 +1,29 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.8.5"
+julia_version = "1.9.3"
```

Comment:
It's generally safer to use the oldest supported version of Julia to generate the manifest.